docker/Dockerfile.nvidia

```diff
@@ -17,22 +15,19 @@
 #   docker build --network=host --build-arg MPI_VER=4 --file docker/Dockerfile.nvidia --tag devito:nvidia .
 #
 # RUN: 
-#   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devito:nvidia
-#   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 --device=/dev/infiniband/uverbs0 --device=/dev/infiniband/rdma_cm  devito:nvidia 
-#
-# to run in user context on a cluster with shared filesystem, you can add the correct user config as docker options e.g.:
-#   docker run --gpus all --rm -it -v `pwd`:`pwd` -w `pwd` -u $(id -u):$(id -g) devito.nvidia python examples/seismic/acoustic/acoustic_example.py
+#   docker run --gpus all --rm -it devito:nvidia
```

Comment:
Why are the ports not needed anymore?

Reply:
The Jupyter server was moved to `docker/Dockerfile.nvidia.nvc`
https://github.com/devitocodes/devito/pull/1797/files#diff-7048a3d844c4c053a5019895477411829650930e75dd2591d147c72ee3e2efbaR39